.github/workflows/makefile.yml

```diff
@@ -0,0 +1,25 @@
+
+name: Make F3DEX3 Binary File.
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: sudo apt install cmake build-essential
+
+    - name: Get armips
+      run: git clone --recursive https://github.com/Kingcom/armips.git ; cd armips ; mkdir build && cd build ; cmake -DCMAKE_BUILD_TYPE=Release .. ; cmake --build . ; sudo cp armips /usr/bin
+
+    - name: Make The Binary
+      run: make -j
```

Comment:
with just `make -j ARMIPS=armips/build/armips` it will work without needed to copy it in /usr/bin